### PR TITLE
Update 3.1.1-install-instructions.md

### PR DESCRIPTION
**Pull request by ebergstedt**
The correct syntax is --classic for snap

### CODE REVIEW

File: release-notes/3.1/3.1.1/3.1.1-install-instructions.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ Snap is a system which installs applications in an isolated environment and prov
 
 After configuring Snap on your system, run the following command to install the latest .NET Core SDK.
 
-`sudo snap install dotnet-sdk --channel 3.1/stable –classic`
+`sudo snap install dotnet-sdk --channel 3.1/stable -–classic`
```
Comment (omajid) [inline release-notes/3.1/3.1.1/3.1.1-install-instructions.md:23]: The second `-` in `--classic` is actually a `–` (em-dash?) here. That might confuse anyone who copy/pastes this code. Maybe change both dashes to `-`?

**Comment by mairaw** [comment]
@ebergstedt can you address @omajid's comment?

File: release-notes/3.1/3.1.1/3.1.1-install-instructions.md (hunk 1 above)
Comment (rbhanda) [reply release-notes/3.1/3.1.1/3.1.1-install-instructions.md]: I have updated the command to `--classic`

**Comment by mairaw** [comment]
Thanks!

**Pull request merged**